tests/e2e/conftest.py

```diff
@@ -16,7 +16,7 @@
 
 @pytest.fixture(scope="session")
 def fxt_ci_data_root() -> Path:
-    data_root = Path(os.environ.get("CI_DATA_ROOT", "/home/validation/data"))
+    data_root = Path(os.environ.get("CI_DATA_ROOT", "/local/otx-ci/ci_datasets"))
```

Comment:
please do not change the default value of the CI_DATA_ROOT for your preference. if you want to set it to your local path, set it through the environment variable.
e.g., 
> CI_DATA_ROOT="/local/otx-ci/ci_datasets" tox -e e2e-test-xxx

Reply:
Yes. I'm running it like it. Is there any special reason to keep the default path to /home/validation/data?